### DISCUSSION TITLE
Chest trap in crypt hitting multiple times depending on what direction it's opened from

### DISCUSSION THREAD

**Discussion by Taohn** [diasurgical/DevilutionX#1671] [Q&A] [answered]
It's a small chest with a fire arrow trap (the one that launches from the chest itself rather than the wall). Normally it deals 10-15 damage or so with a single projectile, but around half the time the character will be hit multiple times in quick succession (several of which are blocked), dying almost instantly. It can happen from any direction but seems to do so most often from the position in the attached save. [single_1.zip](https://github.com/diasurgical/devilutionX/files/6368945/single_1.zip)

I haven't been able to find any reference to anything similar online, so I just wanted to ask if this was a bug or a feature. I've had the same thing happen with nova traps, but that at least makes some kind of sense since the projectiles can overlap.

Thanks for your great work!

**Comment by AJenbo** [comment]
This is usually referred to as the elemental bug, and it's in the original game also. Could you please file this as a bug instead of a discussion?

**Comment by mewmew** [comment, marked as answer]
Was this bug reported as an issue? If so, what issue number is tracking it? :)

**Reply by julealgon** [reply]
https://github.com/diasurgical/devilutionX/issues/1672

**Reply by mewmew** [reply]
perfect, thanks!
